src/EmployeeBook.java

```diff
@@ -0,0 +1,189 @@
+public class EmployeeBook {
+    private Employee[] employees;
+    public EmployeeBook(){
+        this.employees = new Employee[10];
+        initializeEmployees();
+    }
+    private void initializeEmployees(){
+        employees [0] = new Employee("Иванов", "Иван", "Иванович", 25180, 4);
+
+        employees [1]  = new Employee("Сидоров", "Павел", "Петрович", 70350, 1);
+
+        employees [2] = new Employee("Плашкин", "Андрей", "Леонидович", 48320, 3);
+        employees [3] = new Employee("Никитин", "Павел", "Леонидович", 47920, 1);
+        employees [4] = new Employee("Сухина", "Светлана", "Леонидовна", 125020, 3);
+        employees [5] = new Employee("Степкина", "Анжела", "Павловна", 108320, 2);
+        employees [6] = new Employee("Добрышев", "Андрей", "Георгиевич", 43751, 2);
+        employees [7] = new Employee("Дроздов", "Сергей", "Анатольевич", 48805, 1);
+        employees [8] = new Employee("Зырянова", "Наталья", "Аркадьевна", 56378, 5);
+        //employees [9] = new Employee("Плашкин", "Сергей", "Леонидович", 50240, 4);
+        }
+
+
+    public void printAllEmployees() {
+        for (Employee e : employees) {
+            if (e != null) {
+                System.out.println(e);
+            }
+        }
+    }
+
+    public double printSumSalaryEmployees() {
+        if (employees == null || employees.length == 0) {
+            return 0.0;
+        }
+        double totalSalary = 0;
+        for (Employee e : employees) {
+            if (e != null) {
+                totalSalary = totalSalary + e.getSalary();
+            }
+        }
+        return totalSalary;
+
+    }
+
+    public Employee printMinSalaryEmployees() {
+        if (employees == null || employees.length == 0) {
+            return null;
+        }
+        Employee minSalaryEmployee = employees[0];
+        double minSalary = employees[0].getSalary();
```

Comment:
если в массиве нет сотрудника на первом месте, то здесь будет исключение. Лучше положить в переменную макс возможное значение

```suggestion
        double minSalary = Double.MAX_VALUE;
```